app/src/NativeBase/Components/List.tsx

```diff
@@ -116,18 +130,62 @@ const List: FC<ListProps> = ({ data, mode, options, searchScreen, type }) => {
       return (
         <FlatList
```

Comment:
The one last thing I've spotted is that the bottom of the last item in the list is being cut off.  

I think (but am not certain) that you might need to set a `maxHeight` for the `FlatList` using an approach similar to what's being done in [ProjectRegisterInterest](https://github.com/Scottish-Tech-Army/Volunteer-app/blob/main/app/src/NativeBase/Components/Project/ProjectRegisterInterest.tsx) -- define a height for the stuff that's above the list (like line 37), calculate the height of the window (line 158) , then set a `maxHeight` of one minus the other (like line 171 except that uses `minHeight` whereas I think you want to use `maxHeight` here).

Reply:
thank you for spotting, that's an embarrassing miss on my part! 😄 messed about with it there and it is kind of working but there's a bit of white space above the nav icons now...not sure if that's okay or not 

(red line = empty space)

<img width="324" alt="Screenshot 2023-04-18 at 20 57 27" src="https://user-images.githubusercontent.com/87582782/232891694-ca0fa7ff-cb29-4341-9398-e3c0a43f7644.png">

Reply:
@CatMac228 that's looking cut off on my emulator?

![Screenshot 2023-04-19 10 26 14](https://user-images.githubusercontent.com/25086222/233032483-50b3cbf6-05cb-4e82-88b8-6b1a1f9c8e25.png)

Reply:
oh no it looks so different on IOS 😢
<img width="371" alt="Screenshot 2023-04-19 at 12 15 03" src="https://user-images.githubusercontent.com/87582782/233059758-c182d640-1b09-4763-98fb-8d8205921211.png">

Reply:
they seem to be behaving the same way when I use minHeight but not maxHeight 🤔

Reply:
It's possible I'm wrong about `maxHeight` and it should be `minHeight` -- was just a suggestion off the top of my head.  If you've got a version that works do push again and I can try it my end too.  I'm travelling this afternoon but should be able to get online again this evening.

Reply:
No probs! Will remove the maxHeight bit for now although the list will be cut off slightly at the bottom. Wonder if it could be related to these issues https://github.com/facebook/react-native/issues/23693 &https://github.com/facebook/react-native/issues/36447 ?